.templates/gpu.sh.template

```diff
@@ -14,14 +29,9 @@
         (
             flock -x -w 540 200 || exit 1
             export CUDA_VISIBLE_DEVICES="$GPU_NUM"
-            export TF_FORCE_GPU_ALLOW_GROWTH=true
             pytest $TEST_ARGS nengo_dl/tests/test_benchmarks.py::test_performance --performance -v --durations 20 --color=yes || exit 1
+            export TF_FORCE_GPU_ALLOW_GROWTH=true
```

Comment:
Do we need this at all? What's the problem with taking over the whole GPU?

Reply:
It's because we're running the tests in two processes to speed things up (so those processes need to be able to share the GPU).